facia/app/services/dotcomrendering/FaciaPicker.scala

```diff
@@ -16,6 +16,9 @@ object FrontChecks {
   def hasNoEmailCollections(faciaPage: PressedPage) =
     !faciaPage.collections.exists(collection => EmailLayouts.all.contains(collection.collectionType))
 
+  def hasNoShowcaseCollections(faciaPage: PressedPage) =
+    !faciaPage.collections.exists(collection => collection.collectionType == "fixed/showcase")
```

Comment:
Maybe worth pointing to other bits of code where this is used?

```suggestion
  /** @see ShowcaseSingleStories */
  def hasNoShowcaseCollections(faciaPage: PressedPage) =
    !faciaPage.collections.exists(collection => collection.collectionType == "fixed/showcase")
```

Reply:
I've decided against this as it actually doesn't have a reference to this string so might be more confusing.

It might be good to find out where the source for this data it and validate against that going forward.